dbt_airflow_factory/k8s/k8s_parameters.py

```diff
@@ -56,6 +56,7 @@ def __init__(
         secrets: Optional[List[Secret]] = None,
         is_delete_operator_pod: bool = True,
         config_file: str = "~/.kube/config",
+        execution_script: bool = False,
```

Comment:
that should be script name, not boolean. We want user to define their own scripts if needed.

Reply:
Changed: 348912fe75fb1ff0213845d4de02a3d697a55bc2